tests/xhamster-us.spec.ts

```diff
@@ -0,0 +1,5 @@
+import generateCMPTests from "../playwright/runner";
+
+generateCMPTests('xhamster-us', [
+  'https://xhamster.com/',
+], {});
```

Comment:
Note that you can indicate a region here, so our test runner only attempts from specific regions. E.g. `onlyRegions: ['US']` or `excludeRegions: ['FR']`. Not insisting on this though, since we don't block PRs on this, but JFYI